### ISSUE TITLE
Error trying to notify

### ISSUE THREAD

**Issue by daurpam** [Tautulli/Tautulli#628]
The log show this, when try to notify in slack. Don't know what really happens, but I suppose, plexpy is trying to send a "Playback Start" notification, that can't send. Excuse my data expose is the first time I post an issue.

```
Uncaught exception: Traceback (most recent call last):File "/opt/plexpy/plexpy/logger.py", line 221, in new_run old_run(*args, **kwargs)
File "/usr/lib/python2.7/threading.py", line 754, in run self.__target(*self.__args, **self.__kwargs)
File "/opt/plexpy/plexpy/notification_handler.py", line 55, in notify agent_id=agent['id'])
File "/opt/plexpy/plexpy/notification_handler.py", line 639, in build_notify_text poster_url = helpers.uploadToImgur(os.path.join(plexpy.CONFIG.CACHE_DIR, 'cache-poster.jpg'), poster_title)
File "/opt/plexpy/plexpy/helpers.py", line 551, in uploadToImgur response = urllib2.urlopen(request)
File "/usr/lib/python2.7/urllib2.py", line 154, in urlopen return opener.open(url, data, timeout)
File "/usr/lib/python2.7/urllib2.py", line 431, in open response = self._open(req, data)
File "/usr/lib/python2.7/urllib2.py", line 449, in _open '_open', req)
File "/usr/lib/python2.7/urllib2.py", line 409, in _call_chain result = func(*args)
File "/usr/lib/python2.7/urllib2.py", line 1240, in https_open context=self._context)
File "/usr/lib/python2.7/urllib2.py", line 1197, in do_open raise URLError(err)
URLError: 
```

Extra info:
Version plexpy: 1.3.11
Platform: Linux 3.10.77 (Synology DS716+, DSM 6.0-7321 with Docker 1.9.1-0185
Python Version: 2.7.11 (default, Dec 15 2015, 16:46:19) [GCC 4.8.4]

**Comment by JonnyWong16** [comment]
I think I fixed this on the dev branch in 52361cd505c9b17cc138ab07cca8111dbab5be2a

Can you post the entire log file to [pastebin](http://pastebin.com/)?

**Comment by daurpam** [comment]
Hope it helps

[http://pastebin.com/qUXJW5pt](http://pastebin.com/qUXJW5pt)

**Comment by JonnyWong16** [comment]
Are you using posters in your notifications?

**Comment by daurpam** [comment]
Yes

**Comment by daurpam** [comment]
Sorry, answer to fast, only in facebook notify, not in slack notifications

**Comment by JonnyWong16** [comment]
Basically PlexPy just failed to upload the poster to Imgur. Comment again if it happens again.

I found another bug which needs to be fixed.

**Comment by daurpam** [comment]
Ok, thanks for your fast response. And I take this opportunity to thank your work!!

**Comment by JonnyWong16** [comment]
Should be fixed in v1.3.12.
